Source/WebKit/GPUProcess/graphics/WebGPU/RemoteBuffer.cpp

```diff
@@ -96,7 +99,8 @@ void RemoteBuffer::unmap(Vector<uint8_t>&& data)
 
 void RemoteBuffer::destroy()
 {
-    m_backing->destroy();
+    if (m_objectHeap.removeObject(m_identifier))
+        m_backing->destroy();
```

Comment:
Won't this happen automatically?

Reply:
because m_backing's destructor should run? Buffer::Buffer's destructor doesn't call destroy, maybe that is fine, but who calls Buffer::destroy() then?